src/base/packer.cpp

```diff
@@ -922,8 +922,8 @@ namespace cubpacking
   packer::delegate_to_or_buf (const size_t size, or_buf &buf)
   {
     check_range (m_ptr, m_end_ptr, size);
-    m_ptr += size;
     OR_BUF_INIT (buf, m_ptr, size);
+    m_ptr += size;
```

Comment:
The position to move m_ptr has been moved after OR_BUF_INIT() is specified. 
Does that mean the code before the change was a bug?

Reply:
Yes, this function (`packer::delegate_to_or_buf`) is defined to migrate with legacy OR code. But the function was not used anywhere. After testing it for un-implemented float and double types, I found a bug. The ptr should be move after OR_BUF is initialized.